asgs_main.sh

```diff
@@ -2843,8 +2850,8 @@ while [ true ]; do
       # nowcast directory; therefore, the non-existence of the nowcast
       # directory is evidence that something has gone wrong in prep
       if [[ ! -d $NOWCASTDIR ]]; then
-   	CURRENT_EVENT="REND"
-	   CURRENT_STATE="CMPL"
+   	     CURRENT_EVENT="REND"
```

Comment:
what's the purpose of this white space? They should lined up with everything else in the current block